cmd/state-offline-installer/runInstall.go

```diff
@@ -142,7 +148,17 @@ func runInstall(out output.Outputer, params *Params) error {
 	}
 	out.Print(fmt.Sprintf("Stage 3 of 3 Finished: Installing artifacts from: %s", artifactsPath))
 
-	configureEnvironmentAccepted, err := prompt.Confirm("Setup", "Setup environment for installed project?", &default_boolean_answer)
+	// Copy license file
+	contents, err := assets.ReadFileBytes(TOSPath)
```

Comment:
https://github.com/ActiveState/cli/pull/1964#discussion_r951973863

Reply:
Also note that when you use embed you shouldn't need to write this file to disk at all. Its bytes will be in memory for you to access.

If we need the license file in the installed dir then that should IMO be part of the runtime itself, not something we bake into it after the fact.